src/Common/src/Interop/Unix/libssl/Interop.OpenSsl.cs

```diff
@@ -307,45 +302,40 @@ private static void SetSslCertificate(libssl.SafeSslContextHandle contextPtr, Sa
             Debug.Assert(keyPtr != null && !keyPtr.IsInvalid, "keyPtr != null && !keyPtr.IsInvalid");
 
             int retVal = libssl.SSL_CTX_use_certificate(contextPtr, certPtr);
-            if (1 != retVal)
-            {
-                throw CreateSslException("Failed to use SSL certificate");
-            }
+
+            Debug.Assert(1 == retVal, "Failed to use SSL certificate.");
 
             retVal = libssl.SSL_CTX_use_PrivateKey(contextPtr, keyPtr);
-            if (1 != retVal)
-            {
-                throw CreateSslException("Failed to use SSL certificate private key");
-            }
+
+            Debug.Assert(1 == retVal, "Failed to use SSL certificate private key.");
 
             //check private key
             retVal = libssl.SSL_CTX_check_private_key(contextPtr);
-            if (1 != retVal)
-            {
-                throw CreateSslException("Certificate pivate key check failed");
-            }
+
+            Debug.Assert(1 == retVal, "SSL certificate pivate key check failed.");
```

Comment:
Same question about why these have been changed from exceptions to asserts.  I'll stop asking about such things now.